package/MDAnalysis/analysis/dssp/dssp.py

```diff
@@ -323,6 +324,12 @@ def __init__(
             for t in heavyatom_names
         }
         self._donor_mask: Optional[np.ndarray] = ag.residues.resnames != "PRO"
+        self._box = self._trajectory.ts.dimensions
+        if np.array_equal(self._box, (1, 1, 1, 90, 90, 90)):
+            self._box = None
+            warnings.warn(
+                "Box dimensions are (1, 1, 1, 90, 90, 90), not using periodic boundary conditions in DSSP calculations"
```

Comment:
Not 100% on what the warning should be - but this is how we are imagining it? @IAlibay

Reply:
Yes that looks about right. With regards to the warning, your message is close enough to what I was thinking, i.e. something like "Default unit cells of 1 A cubed found. This likely comes from a cryo-em file. To avoid usses, we are disabling periodic boundary conditions in the DSSP calculation".

Reply:
Yeah this is what I was trying to convey earlier in the conversation when I was saying that the PDB reader takes care of that edge case already. I'm not sure what the best approach is here, thoughts @marinegor ?